SimGeneral/MixingModule/python/digitizers_cfi.py

```diff
@@ -84,7 +84,28 @@
 from SimFastTiming.Configuration.SimFastTiming_cff import mtdDigitizer
 from Configuration.Eras.Modifier_phase2_timing_layer_cff import phase2_timing_layer
 phase2_timing_layer.toModify( theDigitizers,
-                              fastTimingLayer = mtdDigitizer.clone() )
+                              fastTimingLayer = mtdDigitizer.clone(),
+                              MC_fastTimingLayer = mtdDigitizer.clone(
+    accumulatorType   = cms.string("MtdTruthAccumulator"),
+    MinEnergy = cms.double(0.5),
+    MaxPseudoRapidity = cms.double(5.0),
+    premixStage1 = cms.bool(False),
+    maximumPreviousBunchCrossing = cms.uint32(0),
+    maximumSubsequentBunchCrossing = cms.uint32(0),
+
+    simHitCollections = cms.PSet(
+        btl = cms.VInputTag(
```

Comment:
Is this acronym correct?

Reply:
since both Barrel and Endcap is handled here, I would say that ```btl``` sounds misleading. @AuroraPerego what about ```mtdCollections```?